spec/components/game_board_component_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe GameBoardComponent, type: :component do
+  #pending "add some examples to (or delete) #{__FILE__}"
+  let(:game) {
+    Game.new(passcode: %w(red red green green).to_json)
+  }
+  let(:attempts) {
+    [
+      Attempt.new(values: %w(blue yellow orange purple)),
+      Attempt.new(values: %w(blue yellow orange red))
+    ]
+  }
+  let(:feedback) { GameBoardComponent.new(game: game, attempts: attempts )}
```

Comment:
Either `board` or `game_board` would be better names to represent the concept of this component, not `feedback`.